iceberg/openhouse/internalcatalog/src/test/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalTableOperationsTest.java

```diff
@@ -106,8 +106,9 @@ void testDoCommitAppendSnapshotsInitialVersion() throws IOException {
 
       Map<String, String> updatedProperties = tblMetadataCaptor.getValue().properties();
       Assertions.assertEquals(
-          4,
-          updatedProperties.size()); /*location, lastModifiedTime, version and appended_snapshots*/
+          5,
```

Comment:
`write.parquet.compression-codec` why is this showing up as part of iceberg version upgrade?

Reply:
It is explicitly set by iceberg in 1.5.2: https://github.com/apache/iceberg/blob/5e139f48505b9b622e2cf389eae2bb74d6420510/core/src/main/java/org/apache/iceberg/TableMetadata.java#L93